packages/react-storage/src/components/StorageBrowser/displayText/libraries/en/locationDetailView.ts

```diff
@@ -1,10 +1,45 @@
 import { DEFAULT_LIST_VIEW_DISPLAY_TEXT } from './shared';
 import { DefaultLocationDetailViewDisplayText } from '../../types';
 
+export const DEFAULT_ERROR_MESSAGE = 'There was an error loading items.';
+
 export const DEFAULT_LOCATION_DETAIL_VIEW_DISPLAY_TEXT: DefaultLocationDetailViewDisplayText =
   {
     ...DEFAULT_LIST_VIEW_DISPLAY_TEXT,
-    getListResultsMessage: () => 'help me',
+    getListItemsResultMessage: (data) => {
+      const {
+        items,
+        query,
+        hasExhaustedSearch,
+        hasError = false,
+        message,
+      } = data ?? {};
+
+      if (hasError) {
+        return {
+          type: 'error',
+          content: message ?? DEFAULT_ERROR_MESSAGE,
+        };
+      }
+
+      if (!items?.length) {
+        return {
+          type: 'info',
+          content: 'No files.',
+        };
+      }
+
+      if (query && hasExhaustedSearch) {
+        return {
+          type: 'warning',
+          content: `No more folders or files found by ${query}.`,
```

Comment:
```suggestion
          type: 'info',
          content: `Showing results for up to the first 10,000 items`,
```

We can remove `searchExhaustedMessage ` below.